FogCreek/Glitch-Community
# various-dart: fix shared-components CSS dominance

## Links
* https://various-dart.glitch.me
* https://app.clubhouse.io/glitch/story/3820/investigate-shared-components-css-dominance

## GIF/Screenshots:

## Changes:
* Move app CSS to top of `<body>` to fix cascade issues
* spent a lot of time looking into react hydration, am pretty sure that we're not actually using it in the case of the CSS files (they're not in helmet)

## How To Test:
* Honestly nothing should have changed.
* I'd recommend throttling, and make sure that you don't get giant icons on page load (shows that we're still properly pulling in shared-components CSS for SSR rendering)
* You can also check the page source and/or the HTML in the devtools for proof that app CSS is always showing up after shared-components `<style>` tags?

## Feedback I'm looking for:
Do you have a better way to do this?

## Things left to do before deploying:
- [ ] update shared-components for a couple global overrides I hadn't added in previously
